webpack.config.js

```diff
@@ -51,6 +51,7 @@ module.exports = (env, argv) => {
 
       // Electron:
       '@electron/remote': '@electron/remote',
+      electron: 'commonjs2 electron',
```

Comment:
We already remove `electron` from the bundle with `alias: { electron: false }`

Reply:
Then I don't think we understand that mongosh failure 😞

Reply:
I filed a ticket. Will close this https://jira.mongodb.org/browse/VSCODE-427